labs/config/settings.py

```diff
@@ -66,3 +66,5 @@
 
 REDIS_HOST = os.environ.get("REDIS_HOST")
 REDIS_PORT = os.environ.get("REDIS_PORT")
+
+LOCAL_LLM = eval(os.environ.get("LOCAL_LLM", "False"))
```

Comment:
Newbie question: why do we need the eval here?

Reply:
`os.environ.get` always returns a string, so instead of the boolean `True` or `False` it would be a string `"True"` or `"False"`